src/components/inlineEdit/js/InlineEdit.js

```diff
@@ -353,8 +353,9 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             editModeInstruction.show();
 
             var editFieldPosition = editField.offset();
+            var editFieldHeight = parseInt(editField.css("height"), 10);
```

Comment:
Can you add a code comment with the jQuery height docs to explain why `css("height")` is preferred.